src/fairplay.js

```diff
@@ -10,7 +10,8 @@ import window from 'global/window';
 import {stringToUint16Array, uint16ArrayToString, getHostnameFromUri, mergeAndRemoveNull} from './utils';
 import {httpResponseHandler} from './http-handler.js';
 
-export const FAIRPLAY_KEY_SYSTEM = 'com.apple.fps.1_0';
+export const FAIRPLAY_KEY_SYSTEM = 'com.apple.fps';
```

Comment:
Is this value used anywhere after this change?

Reply:
It is not, and I was actually intending to remove this constant for that reason and missed it. Code reviews FTW!